en/docs/api-design-manage/design/create-api/create-streaming-api/create-a-websub-streaming-api.md

```
@@ -73,33 +73,52 @@ Follow the instructions below to create a WebHook API using the basic flow:
+
+    2. Select **receive** under **Types**, enter the **Address** of the channel to be created, enter the **Operation Name** to be added to the selected channel, and click **+** as shown below, to add a new topic.
+
+         <a href="{{base_path}}/assets/img/design/create-api/streaming-api/websub-api-v3-add-topic.png"><img src="{{base_path}}/assets/img/design/create-api/streaming-api/websub-api-v3-add-topic.png" width="80%" alt="WebSub API Add Topic"></a>
+
+         The newly added topic is displayed as follows. Expand the topic to view the newly added operation.
+
+         <a href="{{base_path}}/assets/img/design/create-api/streaming-api/websub-api-v3-newly-added-topic.png"><img src="{{base_path}}/assets/img/design/create-api/streaming-api/websub-api-v3-newly-added-topic.png" width="80%" alt="WebSub API Newly Added Topic"></a>
 
          Expand the created topic to view its **Callback URL**. You can use this when registering the topic with a WebHook provider (WebSub Provider).
 
+     3. You can also add a new operation to an existing topic by selecting the respective channel address from the dropdown.
```

markdownlint-cli2 (0.22.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 4; Actual: 5

(MD005, list-indent)